PocketKit/Sources/PocketKit/Article/ReadableViewModel/SavedItemViewModel.swift

```diff
@@ -145,12 +149,12 @@ extension SavedItemViewModel {
         ]
     }
 
-    private func favorite() {
+    internal func favorite() {
```

Comment:
```suggestion
    func favorite() {
```

```suggestion
    func favorite() {
```

Internal by default; no need to explicitly mark it as such.